.babelrc

```diff
@@ -4,6 +4,7 @@
     "@babel/preset-react"
   ],
   "plugins": [
+    "emotion",
```

Comment:
move to last

Reply:
NVM now when i know what it is (compatitor of styled componenets) get rid of it...
find other lib for loading like MUI or something else

Reply:
LOL, are U serious?

Reply:
Yep..
We don't need conflicts of same libs

Reply:
.